examples/app-basic/.gitignore

```
@@ -29,4 +29,13 @@ e2e-results/
 # Editor
 .idea
 
-.eslintcache
\ No newline at end of file
+.eslintcache
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/playwright/.auth/
```

Same suggestion as root gitignore:

```suggestion
/blob-report/
/playwright-report/
/playwright/
/test-results/
```